qiskit_nature/second_q/algorithms/excited_states_solvers/qeom.py

```diff
@@ -503,11 +588,6 @@ def _build_commutator_routine(
 
         eom_operators = {"q": q_mat_op, "w": w_mat_op, "m": m_mat_op, "v": v_mat_op}
 
-        if not z2_symmetries.is_empty():
-            for index_op, eom_op in eom_operators.items():
-                if eom_op is not None and len(eom_op) > 0:
-                    eom_operators[index_op] = z2_symmetries.taper_clifford(eom_op)
```

Comment:
Why can we remove this?

Reply:
These lines are basically reimplementing `symmetry_reduce_clifford()`. They are contained inside a private method so I opted for a SparsePauliOp output without the symmetry reduction. Later on, they are converted to PauliSumOp and `symmetry_reduce_clifford()` is applied just before we pass them to `estimate_observables()`